queries.sql

```diff
@@ -10,3 +10,48 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE NOT name = 'Gabumon';
 SELECT * FROM animals WHERE weight_kg BETWEEN  10.4 AND 17.3;
+
+BEGIN;
+UPDATE animals
+SET species = 'unspecified';
+SELECT * FROM animals;
+ROLLBACK;
+
+BEGIN;
+UPDATE animals
+SET species = 'digimon'
+WHERE name LIKE '%mon';
+SELECT * FROM animals;
+COMMIT;
+SELECT species from animals;
+
+BEGIN;
+UPDATE animals
+SET species = 'pokemon'
+WHERE species IS null;
+SELECT * FROM animals;
+COMMIT;
```

Comment:
- [ ]  In accordance to the project requirement, we are to verify if the changes persist after "`SET species = 'pokemon'`", to achieve this I think you can use the select query to check as suggested earlier. 🙏

- [optional] Kindly update the screenshot in the PR as stated in the requirement. Thanks 

<img width="715" alt="Untitled" src="https://github.com/Kingno3l/Vet-Clinic/assets/103386183/1314142a-416d-4656-8659-b25aa8a059ac">